src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,140 @@
+package Duke;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Duke {
+    //Constants
+    private static final String MESSAGE_BOUNDARY = "____________________________________________________________";
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    //class variables
+    private List<Task> myTasks;
+
+    //Constructor for class Duke.Duke
+    public Duke() {
```

Comment:
This may count as excessive use of the this word?
https://se-education.org/guides/conventions/java/index.html#variables 
https://github.com/nus-cs2113-AY2021S1/forum/issues/27